ij-plugin/src/main/resources/org/jetbrains/research/tasktracker/config/survey_default.yaml

```diff
@@ -1,210 +1,119 @@
 surveys:
   - id: default
     htmlQuestions:
-      - !<Input>
-        text: Approximately how many hours did you use the plugin?
-        elementId: hours
-        type: number
+      - !<Textarea>
+        text: 1.How helpful was the hint in solving the problem?
```

Comment:
Could you please add a clarification to the question:  1 - not helpful at all, 5 - very helpful or something like this